### DISCUSSION TITLE
Permit styling the cursors that are used by pickers and prompts.

### DISCUSSION THREAD

**Discussion by 7ombie** [helix-editor/helix#10435] [General]
Themes cannot currently assign a color to the cursors used in pickers and prompts.

If, for example, you want your secondary cursors to be the same color as the cursors in pickers and prompts, you must configure your terminal to use the same color for *its* cursors as your Helix theme assigns to cursors for secondary selections.

Helix allows users to assign colors to the three primary modes, then use the corresponding color to highlight the mode indicator and cursors. I personally use this feature to color-code the mode indicator and primary cursor (Insert is green, Normal is yellow and Select is red), then use blue for any secondary cursors. Having the cursors in pickers and prompts use the same blue as secondary cursors reinforces the associations and distinctions. However, I couldn't ship something like this as a theme (without requiring the user to configure their terminal a certain way).

**Comment by mesmere** [comment]
Yeah it picks up the default cursor color from the terminal. This is a real pain for me bc I use a black terminal and white helix so the cursor blends in to one or the other one.

![image](https://github.com/user-attachments/assets/9f0a2b14-4c39-4391-ad78-2245596f9274)

![image](https://github.com/user-attachments/assets/b229d455-d7b1-4ae6-8e43-212190b6dd88)
